crates/sui-benchmark/src/workloads/data/adversarial-dynamic-fields/sources/adversarial_dynamic_fields.move

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Could you put this as an additional module in the `adversarial` package rather than creating an extra package? That will save us from having to do an extra publish command, remembering an extra package ID, etc.

Although: I don't actually see the extra publish command, so unsure how this is working right now...

Reply:
Ah you can ignore this file. It's WIP and made it in accidentally.
The reason for this new package was that we were running out of gas in the module initializer of the original package, so we could not create enough dynamic fields. My hope was that if we publish a smaller package just for dynamic field testing, we can have more gas left for the initializer, but this is WIP.